packages/kit/src/views/Send/pages/SendDataInput/SendDataInputContainer.tsx

```
@@ -4,7 +4,7 @@ import { memo, useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { useRoute } from '@react-navigation/core';
 import BigNumber from 'bignumber.js';
 import { utils } from 'ethers';
-import { isNaN, isNil } from 'lodash';
+import { isEmpty, isNaN, isNil } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)